diff --git a/node_modules/@discord-player/equalizer/dist/index.js b/node_modules/@discord-player/equalizer/dist/index.js
index 19fea66..549acd8 100644
--- a/node_modules/@discord-player/equalizer/dist/index.js
+++ b/node_modules/@discord-player/equalizer/dist/index.js
@@ -1551,14 +1551,14 @@ var _FiltersChain = class _FiltersChain {
   create(src, presets = this.presets) {
     this.destroy();
     this.source = src;
-    const resampler = !presets.resampler?.disabled ? new PCMResampler(presets.resampler) : null;
-    const equalizerStream = !presets.equalizer?.disabled ? new EqualizerStream(presets.equalizer) : null;
-    const dspStream = !presets.dsp?.disabled ? new AudioFilter(presets.dsp) : null;
-    const biquadStream = !presets.biquad?.disabled ? new BiquadStream(presets.biquad) : null;
-    const volumeTransformer = !presets.volume?.disabled ? new VolumeTransformer(presets.volume) : null;
-    const compressor = !presets.compressor?.disabled ? new CompressorTransformer(presets.compressor) : null;
-    const seeker = !presets.seeker?.disabled ? new PCMSeekerTransformer(presets.seeker) : null;
-    const reverb = !presets.reverb?.disabled ? new ReverbTransformer(presets.reverb) : null;
+    const resampler = presets.resampler && !presets.resampler.disabled ? new PCMResampler(presets.resampler) : null;
+    const equalizerStream = presets.equalizer && !presets.equalizer.disabled ? new EqualizerStream(presets.equalizer) : null;
+    const dspStream = presets.dsp && !presets.dsp.disabled ? new AudioFilter(presets.dsp) : null;
+    const biquadStream = presets.biquad && !presets.biquad.disabled ? new BiquadStream(presets.biquad) : null;
+    const volumeTransformer = presets.volume && !presets.volume.disabled ? new VolumeTransformer(presets.volume) : null;
+    const compressor = presets.compressor && !presets.compressor.disabled ? new CompressorTransformer(presets.compressor) : null;
+    const seeker = presets.seeker && !presets.seeker.disabled ? new PCMSeekerTransformer(presets.seeker) : null;
+    const reverb = presets.reverb && !presets.reverb.disabled ? new ReverbTransformer(presets.reverb) : null;
     this.resampler = resampler;
     this.equalizer = equalizerStream;
     this.filters = dspStream;
